src/bin/copy.c

```diff
@@ -79,18 +83,87 @@ static char *getuser(void)
 	return pw->pw_name;
 }
 
+/*
+ * If UNIX user is in UNIX group of directory containing file,
+ * return gid, otherwise -1
+ *
+ * E.g., writing to /cfg/foo, where /cfg is owned by root:wheel,
+ * should result in the file being owned by $LOGNAME:wheel with
+ * 0660 perms for other users in same group.
```

Comment:
If we made sure that we create `startup` with `g+s`, could we rely on the kernel to handle this logic for us?

Reply:
> If we made sure that we create `startup` with `g+s`, could we rely on the kernel to handle this logic for us?

You mean on `/cfg`, I assume. Yes, but I was worrying about bind + overlay mounts, as well as external media with vfat/exfat file systems.